core/chains/evm/txmgr/helpers_test.go

```diff
@@ -1,15 +1,37 @@
 package txmgr
 
-import evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+import (
+	evmclient "github.com/smartcontractkit/chainlink/v2/core/chains/evm/client"
+)
 
-func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EthConfirmer) {
+func SetEthClientOnEthConfirmer(ethClient evmclient.Client, ethConfirmer *EvmConfirmer) {
 	ethConfirmer.ethClient = ethClient
 }
 
-func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EthBroadcaster) {
+func SetResumeCallbackOnEthBroadcaster(resumeCallback ResumeCallback, ethBroadcaster *EvmBroadcaster) {
 	ethBroadcaster.resumeCallback = resumeCallback
 }
 
-func (er *EthResender) ResendUnconfirmed() error {
+func StartInternalOnBroadcaster(eb *EvmBroadcaster) error {
+	return eb.startInternal()
+}
+
+func CloseInternalOnBroadcaster(eb *EvmBroadcaster) error {
+	return eb.closeInternal()
+}
```

Comment:
Can these be methods?
```suggestion
func (eb *EvmBroadcaster) CloseInternal() error {
	return eb.closeInternal()
}
```